doc/man1/openssl-ca.pod.in

```diff
@@ -179,7 +179,8 @@ See L<openssl-format-options(1)> for details.
 =item B<-sigopt> I<nm>:I<v>
 
 Pass options to the signature algorithm during sign operations.
-Names and values of these options are algorithm-specific.
+Names and values of these options are algorithm-specific and
+documented in L<provider-signature(7)>.
```

Comment:
/Signature parameters> ?

Reply:
sorry, don't understand the request...

Reply:
I was suggesting changing the link to include the section within the provider-signature document.

Reply:
fixed

Reply:
May as well change the other file also..

Reply:
done